app/services/oneoffs/remove_fees_from_ecf_contracts.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class RemoveFeesFromECFContracts
+    attr_reader :cohort_year, :from_date
+
+    def initialize(cohort_year:, from_date:)
+      @cohort_year = cohort_year
+      @from_date = from_date.to_date
+    end
+
+    def call
+      statements.each do |statement|
+        lead_provider = statement.lead_provider
+        version = statement.contract_version
+
+        old_contract = CallOffContract.where(
+          version:,
+          lead_provider:,
+          cohort:,
+        ).first!
+
+        # Contract already set to zero
+        next if old_contract.monthly_service_fee.to_s == "0.0"
+
+        new_version = increment_version(version)
+
+        new_contract = CallOffContract.find_or_initialize_by(
```

Comment:
could you `.dup` the contract as there are other attributes we need to set past those?